### PR TITLE
quba 1.4.1

### PR DESCRIPTION
**Pull request by eleanordoesntcode**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

---
I changed the download URL to the GitHub repo, since the site seems to take more time to update (the 1.4.1 version did not exist on the website when I tried)

### CODE REVIEW

File: Casks/q/quba.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask "quba" do
-  version "1.4.0"
-  sha256 "4a30c194d59fc850650bd007a717745d53dd060e26792642f47e5368f373f1bb"
+  version "1.4.1"
+  sha256 "f16c094dbc132da7326b3982d832398454fbadc72ce96a54fb46f25a1a8f3080"
 
-  url "https://quba-viewer.org/files/Quba-#{version}-arm64.dmg"
+  url "https://github.com/ZUGFeRD/quba-viewer/releases/download/v#{version}/Quba-#{version}-universal.dmg"
```
Comment (khipp) [inline Casks/q/quba.rb:5]:
```suggestion
  url "https://github.com/ZUGFeRD/quba-viewer/releases/download/v#{version}/Quba-#{version}-universal.dmg",
      verified: "github.com/ZUGFeRD/quba-viewer/"
```

**Comment by eleanordoesntcode** [comment]
Just noticed the problem, thanks for the fix, I'll squash the commits now

**Comment by khipp** [comment]
Now that we’re using the GitHub repository, we can also replace the `livecheck` URL with the `:url` symbol.

**Comment by eleanordoesntcode** [comment]
Right, I'll add that too

File: Casks/q/quba.rb [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
-  url "https://quba-viewer.org/files/Quba-#{version}-arm64.dmg"
+  url "https://github.com/ZUGFeRD/quba-viewer/releases/download/v#{version}/Quba-#{version}-universal.dmg",
+      verified: "github.com/ZUGFeRD/quba-viewer"
   name "Quba-Viewer"
   desc "Viewer for electronic invoices"
   homepage "https://quba-viewer.org/"
 
   livecheck do
-    url "https://github.com/ZUGFeRD/quba-viewer"
+    url :verified
```
Comment (khipp) [inline Casks/q/quba.rb:12]:
```suggestion
    url :url
```

We are generally using the `:url` symbol with casks (see [documentation](https://docs.brew.sh/Brew-Livecheck#general-guidelines)).

File: Casks/q/quba.rb [hunk 3]
```diff
@@ -1,14 +1,15 @@
 cask "quba" do
-  version "1.4.0"
-  sha256 "4a30c194d59fc850650bd007a717745d53dd060e26792642f47e5368f373f1bb"
+  version "1.4.1"
+  sha256 "f16c094dbc132da7326b3982d832398454fbadc72ce96a54fb46f25a1a8f3080"
 
-  url "https://quba-viewer.org/files/Quba-#{version}-arm64.dmg"
+  url "https://github.com/ZUGFeRD/quba-viewer/releases/download/v#{version}/Quba-#{version}-universal.dmg",
+      verified: "github.com/ZUGFeRD/quba-viewer"
```
Comment (khipp) [inline Casks/q/quba.rb:6]:
```suggestion
      verified: "github.com/ZUGFeRD/quba-viewer/"
```

One last small change to pass the audit.

**Review by khipp — APPROVED** [review]
Thank you, @eleanordoesntcode!

**Pull request merged**